test/bump.test.js

```diff
@@ -78,7 +78,8 @@ tap.test('should create a new git branch', async t => {
   const branchName = `release/${TEST_VERSION}`
 
   t.ok(stubs.runSpawnStub.calledWith('git', ['checkout', '-b', branchName]))
-  t.ok(stubs.runSpawnStub.calledWith('git', ['commit', '-am', TEST_VERSION]))
+  t.ok(stubs.runSpawnStub.calledWith('git', ['add', '-A']))
```

Comment:
all these assertions should be replaced with sinon.assert. these assertions will only give a generic, non-meaningful error message when they fail